ChopChop/Storage/FirebaseDatabase.swift

```diff
@@ -65,14 +65,26 @@ struct FirebaseDatabase {
             .eraseToAnyPublisher()
     }
 
-    func fetchAllUsers() -> AnyPublisher<[User], Error> {
+    func fetchAllUsers(query: String = "") -> AnyPublisher<[User], Error> {
```

Comment:
just a tiny note on an efficiency problem that we cannot solve:
firebase dosent allow us to do search by string on server side so each time we are actually fetching all users and doing front end filtering

Reply:
Yes I think this is the best we can achieve with Firebase restrictions